src/config_parser.cpp

```diff
@@ -161,6 +162,10 @@ void Parameter_t::ParseConfigFile(const char *param_file)
   PhysicalConst::G = 43.0071 * (MassInMsunh / 1e10) / VelInKmS / VelInKmS / LengthInMpch;
   PhysicalConst::H0 = 100. * (1. / VelInKmS) / (1. / LengthInMpch);
 
+  /* Make particles split by default in swift (only relevant for hydro runs) */
+  if(SnapshotFormat == "swiftsim")
```

Comment:
Ugh, I forgot to upload a commit that change this.

Reply:
The latest commit changes the way in which the parameter is set. Before, it would always make it true for swift simulations, since it didn't know whether the parameter was set or not. The new version tests for this, and uses the given value if it is specified. If not and is using swift, it defaults to true.